kubernetes/launch-job.sh

```diff
@@ -38,17 +38,20 @@ usage() {
   echo "  -p, --predictor"
   echo "    Use AMCTS with a predictor network. (A-MCTS-VM)"
   echo "  --predictor-warmstart-ckpt CHECKPOINT"
-  echo "    Name of checkpoint's TF weights directory to use for predictor warmstart."
+  echo "    Path to checkpoint's TF weights directory to use for predictor warmstart."
   echo "  -r, --resume TIMESTAMP"
   echo "    Resume a previous run. If this flag is given, the PREFIX argument"
   echo "    must exactly be match the run to be resumed, and the TIMESTAMP"
   echo "    argument should match the timestamp attached to the name of the"
   echo "    previous run's output directory. The use of the --use-weka flag"
   echo "    must also exactly match that of the previous run."
   echo "  --warmstart-ckpt CHECKPOINT"
-  echo "    Name to checkpoint's TF weights directory to use for warmstarting"
-  echo "    the adversary, e.g., b6c96-s175395328-d26788732 for cp63 or"
-  echo "    kata1-b40c256-s11840935168-d2898845681 for cp505."
+  echo "    Path to checkpoint's TF weights directory to use for warmstarting"
+  echo "    the adversary, e.g.,"
+  echo "    /shared/victim-weights/kata1-b40c256-s11840935168-d2898845681 for "
+  echo "    cp505, e.g.,"
+  echo "    /shared/victimplay/ttseng-avoid-pass-alive-coldstart-39-20221025-175949/models/t0-s545065216-d136760487"
+  echo "    for the cyclic-adversary."
```

Comment:
The following note or similar might help to document what file/directory naming conventions will work:

"Either the weights file should be model.bin.gz, or up to file extension and addition of 'kata1' the directory and weights file name should match."